src/main/java/de/thomas_oster/liblasercut/FloatMinMaxPowerSpeedFrequencyProperty.java

```diff
@@ -88,11 +90,33 @@ public Object getProperty(String name)
     if (MIN_POWER.equals(name)) {
       return (Float) this.getMinPower();
     }
+    else if (SPEED.equals(name)) {
+      return (Float) this.getSpeed();
+    }
     else {
       return super.getProperty(name);
     }
   }
 
+  /**
+   * Must override since parent class only allows 0 < speed < 100
+   * 
+   */
+
+  @Override
+  public void setSpeed(float speed)
+  {
+    speed = speed < 0 ? 0 : speed;
+    speed = speed > 1000 ? 1000 : speed;
```

Comment:
All speed settings for all other lasercutters are in percent, maximum 100. So this one should be the same for consistency. If we support absolute speeds or other units (mm/s, inch per whatever) one day, then this should be one consistent interface for all/most lasercutters, see https://github.com/t-oster/VisiCut/issues/381